packages/@aws-cdk/aws-certificatemanager/lib/dns-validated-certificate.ts

```diff
@@ -109,7 +109,11 @@ export class DnsValidatedCertificate extends cdk.Resource implements ICertificat
     protected validate(): string[] {
         const errors: string[] = [];
         // Ensure the zone name is a parent zone of the certificate domain name
-        if (this.domainName !== this.normalizedZoneName && !this.domainName.endsWith('.' + this.normalizedZoneName)) {
+        if (
```

Comment:
This indentation looks a tad weird to me... how about this?

```ts
        if (!cdk.Token.isUnresolved(this.normalizedZoneName) &&
              this.domainName !== this.normalizedZoneName &&
              !this.domainName.endsWith('.' + this.normalizedZoneName)) {
            errors.push(`DNS zone ${this.normalizedZoneName} is not authoritative for certificate domain name ${this.domainName}`);
        }
```

Reply:
Works with me!

Reply:
Code in the current revision is still the same though 🙂